<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRodadasCadastro</class>
 <widget class="QDialog" name="frmRodadasCadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs/imagens.qrc">
    <normaloff>:/cadastroRodadas.png</normaloff>:/cadastroRodadas.png</iconset>
  </property>
  <widget class="QTableWidget" name="twgListaRodadas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>931</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Modo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbModo">
      <item>
       <property name="text">
        <string>ANA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DMR</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>341</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Título</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtTitulo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>241</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Horário</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtHorario"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>231</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>TG ou Frequência</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtTgFrequencia"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>161</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Criado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCriado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnGravarAlteracoes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Gravar Alterações"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/salvar.png</normaloff>:/salvar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnArquivar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Desativar/Reativar  a Rodada"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/arquivar.png</normaloff>:/arquivar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="autoRepeatDelay">
    <number>304</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSair">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>11</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Sair"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/sair.png</normaloff>:/sair.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>151</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Modificado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtModificado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtId">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnLimparCampos">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Novo Registro"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/zerar.png</normaloff>:/zerar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInserirRodada">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Inserir na Base de Dados"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/PessoaNovo.png</normaloff>:/PessoaNovo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imgs/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
